### PR TITLE
Issue 101

### PR DESCRIPTION
**Pull request by Atihinen**
- Updated documentation for values and validations with examples

### CODE REVIEW

File: examples/4_values.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The `data` argument can be a dictionary, bytes or a file name. Note that the contents of the
+file is read as bytes.
+
+Example usage:
+
+| Keyword | Field | Arguments | Remarks |
+| ------- | ----- | --------- | ------- |
+| Get | /users/1 | data=${my_dict} | |
+| ${bytes}= Encode String to Bytes | ${some_string} | `Encode String to Bytes` is from `String` library |
```
Comment (Atihinen) [inline examples/4_values.md:39]: I think that `Encode String to Bytes` part should be under remarks, currently it's not

image

Comment (asimell) [reply examples/4_values.md]: Fixed

File: examples/5_validations.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+| `GET`  | /users/1 | | | # https://jsonplaceholder.typicode.com/users/1 |
+| `Object` | response body | |
+| `Object` | response body | required=["id", "name"] | | # can have other properties |
+| `GET`  | /users/1 | | | # https://jsonplaceholder.typicode.com/users/1 |
+| `Object` | $.address.geo | required=["lat", "lng"] |
+| `Object` | $..geo | additionalProperties=false | | # cannot have other properties |
+
+## Array
+
+Keyword `Array` checks the expected field as JSON array.
```
Comment (asimell) [inline examples/5_validations.md:128]: Maybe add a remark that they keyword doesn't validate the array _contents_, just that given JSONPath is an array.

Comment (Atihinen) [reply examples/5_validations.md:128]: Excellent point. I'll add that.

File: examples/5_validations.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+Keyword doesn't verify the conent of the array. You can use e.g. [Collections library](https://robotframework.org/robotframework/latest/libraries/Collections.html) to do that.
```
Comment (asimell) [inline examples/5_validations.md:129]:
```suggestion
Keyword doesn't verify the contents of the array. You can use e.g. [Collections library](https://robotframework.org/robotframework/latest/libraries/Collections.html) to do that.
```

**Review by asimell — APPROVED** [review]
LGTM

**Pull request merged**